kickstarts/partials/post/source_setup.ks.erb

```diff
@@ -0,0 +1,56 @@
+www_root="/var/www"
+app_root="$www_root/miq"
+appliance_root="/opt/manageiq/manageiq-appliance"
+
+rm -rf $app_root
+
+git clone <%= @appliance_checkout.remote %> $appliance_root
+pushd $appliance_root
+  git checkout <%= @appliance_checkout.branch %>
+  git reset --hard <%= @appliance_checkout.commit_sha %>
+popd
+
+mkdir -p $app_root
+git clone <%= @manageiq_checkout.remote %> $app_root/vmdb
+pushd $app_root/vmdb
+  git checkout <%= @manageiq_checkout.branch %>
+  git reset --hard <%= @manageiq_checkout.commit_sha %>
+popd
+
+# Symlink extracted repo (manageiq-appliance) to old /var/www/miq/system location.
+ln -vs $appliance_root $app_root/system
+
+# Symlink old lib directory to gems/pending.
+ln -vs $app_root/vmdb/gems/pending $app_root/lib
+
```

Comment:
Yeah, I kept them together because they all use the variables defined at the top of the file.  It doesn't make a difference functionally because it is just text at this point, but it might be tough to edit the file if we are using variables defined in another partial.